deployment-tinkerer/src/main/java/org/wso2/testgrid/deployment/tinkerer/SessionManager.java

```diff
@@ -207,4 +208,30 @@ private static String getInstanceName(String provider, String region, String ins
         }
     }
 
+    /**
+     * Get the instance username that is used to log into the instance depending on the cloud provider.
+     *
+     * @param provider Provider Name
+     * @param region Region of the instance
+     * @param instanceId ID of the instance
+     * @return Username of the instance
+     */
+    private static String getInstanceUser(String provider, String region, String instanceId) {
+
+        Provider infraProvider = null;
+        switch (provider) {
+            case "aws":
```

Comment:
By any chance, can we receive "aws" with uppercase/ camel case here? If so, we may need to check with case.